src/main/java/frc/robot/utils/DashboardDataContainer.java

```diff
@@ -35,6 +36,10 @@ public DashboardDataContainer() {
 				SmartDashboard.getNumber("Shooter/Shooting Velocity Setpoint", 0)));
 		SmartDashboard.putData("Shooter/Enable Tuning",
 				new StartEndCommand(shooter::enableTuning, shooter::disableTuning));
+
+		// Command groups data
+		SmartDashboard.putData("CommandGroups/AutoShoot", new AutoShoot(() ->
+				SmartDashboard.getNumber("Shooter/Shooting Velocity Setpoint", 0)));
```

Comment:
SmartDashboard.getNumber("Shooter/Shooting Velocity Setpoint", 0)));
add it's putNumber